src/main/java/com/archisacademy/jobportal/controller/UserController.java

```diff
@@ -0,0 +1,93 @@
+package com.archisacademy.jobportal.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.archisacademy.jobportal.dto.UserDto;
+import com.archisacademy.jobportal.enums.UserRole;
+import com.archisacademy.jobportal.services.UserService;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping
+    public ResponseEntity<String> createUser(@RequestBody UserDto userDto) {
```

Comment:
we don't need this create method for user since we will have authentication controller